Makefile

```diff
@@ -118,7 +118,7 @@ html: run_ninja
 
 # Build the html output files and additionally run a small webserver for local previews
 serve_html: run_ninja
-	$(JEKYLL_CMD) serve
+	$(JEKYLL_CMD) serve --watch
```

Comment:
I _think_ the reason that I didn't do this previously, was because a lot of the files that you edit need to be "converted" by some of the other Python scripts first, and so you typically need to run `make html` before running `make serve_html`. And so you might get confused if you were using `serve --watch` and you edited a file which needed to be "converted", which Jekyll isn't able to do automatically? (because you wouldn't see any changes when you refreshed your browser)

But of course this is fine if you actually understand what's going on! :slightly_smiling_face:

Reply:
Yes, this doesn't actually result in a live update of the site, presumably because of those other 'conversions'. I made this change as part of a slow-but-steady effort to eventually get live previews working (it gets some skin in the game).